hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientWriteHandler.java

```diff
@@ -60,17 +61,13 @@ public void handle() {
             ready = true;
             return;
         }
-        try {
-            writeBuffer();
-        } catch (Throwable t) {
-            logger.severe("Fatal Error at WriteHandler for endPoint: " + connection.getEndPoint(), t);
-        } finally {
```

Comment:
There is also a behaviour change. The following lines will not be run when an writeBuffer throws Exception.
```
ready = false
registerWrite()
```
Aside from that this means that when `packet.writeTo` or `clientMessage.writeTo` throws a runtime exception connection will be closed. Before it was just logged and consequent packets may be written safely. Since I dont really see, how `writeTo` method can throw RunTimeException, I want to say it is OK, but I am not %100 sure. Leaving comment here to get attention from more people. ;)